bench_runner/plot.py

```diff
@@ -464,8 +467,15 @@ def get_comparison_value(ref, r):
 
     for subplot, ax in zip(subplots, axs):
         ax.set_title(f"Effect of {subplot['name']}")
+        version = tuple(int(x) for x in subplot["version"].split("."))
+        assert len(version) == 2, (
+            "Version config in {subplot['name']}" " should only be major.minor"
+        )
 
         for runner in mrunners.get_runners():
+            runner_is_mapped = runner.nickname in subplot["runner_map"]
+            if subplot["runner_map"] and not runner_is_mapped:
+                continue
```

Comment:
So the idea here is -- if you provide a runner_map, the runner must be mapped to be shown at all?  I guess that makes sense.  Let's document that in the README.

Reply:
Done.